runtime/doc/options.txt

```diff
@@ -5468,9 +5468,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	"-f" is not inside the quotes, because it is not part of the command
 	name.  And Vim automagically recognizes the backslashes that are path
 	separators.
-	For Dos 32 bits (DJGPP), you can set the $DJSYSFLAGS environment
-	variable to change the way external commands are executed.  See the
-	libc.inf file of DJGPP.
 	Under MS-Windows, when the executable ends in ".com" it must be
```

Comment:
(Just as as thought for now, since windows support is not yet merged and we can't readily test) Are lines 5474-5476 relevant still? Does it make sense to support [`command.com`](https://en.wikipedia.org/wiki/COMMAND.COM) or [`4dos.com`](https://en.wikipedia.org/wiki/4DOS)?